src/components/drawer/bl-drawer.ts

```diff
@@ -57,16 +65,42 @@ export default class BlDrawer extends LitElement {
     window?.addEventListener("bl-drawer-open", event => {
       if (event.target !== this) this.closeDrawer();
     });
+    this.resizeDrawerWidth();
+
+    window?.addEventListener("resize", () => this.resizeDrawerWidth());
+    window?.addEventListener("load", () => this.resizeDrawerWidth());
+  }
+
+  disconnectedCallback(): void {
+    super.disconnectedCallback();
+    window?.removeEventListener("resize", () => this.resizeDrawerWidth());
+    window?.addEventListener("load", () => this.resizeDrawerWidth());
   }
 
   updated(changedProperties: PropertyValues<this>) {
     if (changedProperties.has("open")) {
       this.toggleDialogHandler();
     }
+
+    if (changedProperties.has("width")) {
+      this.resizeDrawerWidth();
+    }
   }
 
   private domExistenceSchedule: number;
 
+  private resizeDrawerWidth() {
```

Comment:
I don't think we should allow values below a certain threshold

![Screenshot 2024-04-02 at 13 44 45](https://github.com/Trendyol/baklava/assets/15656271/66962d55-1157-4799-a3b3-e05b6be23280)

Reply:
i added minimum threshold as 100px